app/Http/Controllers/User/BlogController.php

```diff
@@ -23,17 +25,14 @@ public function defineUser() {
      */
     public function index()
     {
-        $blogs = Blog::all();
-        if ($this->defineUser()->can_delete === Blog::IS_TRUE) {
-            $canDelete = true;
-        }else {
-            $canDelete = false;
-        }
+        $blogs = Blog::where('user_id', '=', $this->defineUser()->id)->get();
+        $canDelete = $this->defineUser()->can_delete === Blog::IS_TRUE;
         $canSee = $this->defineUser()->can_see;
-        if($canSee === Blog::IS_TRUE)
+        if ($canSee === Blog::IS_TRUE) {
```

Comment:
why have this format changing ?

Reply:
Dạ em check phpcs thi bị lỗi, em viết lại theo format PSR2.

Reply:
trả lại như cũ (vì ko phải code sửa trong lần này)

Reply:
I fixed

Reply:
I fixed